File: atividade_6/C/jogos.c
#include <stdio.h>

int main()
{
    int jogos = 0;
    printf("Quantos jogos foram vendidos? ");
    scanf("%d", &jogos);
    double VALOR_JOGO = 19.9;
    double total = jogos * VALOR_JOGO;
    double bonus = jogos / 15 > 0 ? total * (jogos / 15 * 0.08) : 0;
    double salarioCatarina = total / 2 + bonus;
    printf("%.2f %.2f %.2f", total, bonus, salarioCatarina);
    return 0;
}

File: atividade_6/java/Jogos.java
package atividade_6.java;

import java.util.Scanner;

public class Jogos {
    public static void main(String[] args) {
        try (var scan = new Scanner(System.in)) {
            System.out.print("Quantos jogos foram vendidos? ");
            var JOGOS_VENDIDOS = scan.nextInt();
            final double VALOR_JOGO = 19.9;
            var total = JOGOS_VENDIDOS * VALOR_JOGO;
            var bonus = JOGOS_VENDIDOS / 15 > 0 ? total * (JOGOS_VENDIDOS / 15 * 0.08) : 0;
            var salarioCatarina = total / 2 + bonus;
            System.out.printf("%.2f %n%.2f %n%.2f", total, bonus, salarioCatarina);
        } catch (Exception e) {
            System.out.println(e);
        }
    }   
}